src/ReactNativeZoomableView.tsx

```diff
@@ -441,6 +486,12 @@ class ReactNativeZoomableView extends Component<
       );
     }
 
+    if (this.props.staticPinPosition) {
+      this._updateStaticPin();
```

Comment:
if we can call this on after a series of `onStaticPinPositionMove ` events, we won't need this here

Reply:
yup!